Update Messages.json

Nunting opened the pull request:
DONE.
More literal translation is needed, but it's not urgent right now. (Tired)

Nunting committed:
Update Messages.json

DONE

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "SERVER_NAME_3V3": "[yellow]⚡ [red]Ranked [purple]3 vs 3 [yellow]⚡",
+    "SERVER_NAME_HEX": "[yellow]⚡ [red]Ranked [green]H E X [yellow]⚡",
+    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#ff00ff]시즌 VIII",
+    "SERVER_DESCRIPTION_2V2": "[#ff00ff]Season VII",
```
But it's season VIII already. :)

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#ff00ff]시즌 VIII",
+    "SERVER_DESCRIPTION_2V2": "[#ff00ff]Season VII",
+    "SERVER_DESCRIPTION_3V3": "[#ff00ff]Season VII",
+    "SERVER_DESCRIPTION_HEX": "[green]new quality level of hex\n[#ff00ff]Season VII",
```
Can you please translate this? Most probably this won't change.

LeoSko commented on ko/Messages.json:
```diff
@@ -260,16 +391,68 @@
     "LOSE_FORMAT": "패배",
     "TIE_FORMAT": "무승부",
+    "FINISH_FORMAT": "#[yellow]@[] of [yellow]@[]",
+    "RATING_CALCULATION_NEXT_TEAM_ELIMINATION_WARNING": "[red]Your rating change will be calculated when some next team will be eliminated",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -260,16 +391,68 @@
+    "START_MATCH_PLAYER_ENTRY_FORMAT": "[white]@[white] [yellow]@[white]",
+    "START_MATCH_TEAM_ENTRY_FORMAT": "[#@]@[white]: @",
+    "SURRENDER_USAGE": "[yellow]/surrender[white] 를 입력하여 항복",
+    "START_FORMAT": "[green]Map [yellow]@\n@\n@",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -260,16 +391,68 @@
+    "START_MATCH_TEAM_ENTRY_FORMAT": "[#@]@[white]: @",
+    "SURRENDER_USAGE": "[yellow]/surrender[white] 를 입력하여 항복",
+    "START_FORMAT": "[green]Map [yellow]@\n@\n@",
+    "START_FORMAT_UNRANKED": "[green]Map [yellow]@\nUnranked match\n@\n@",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -260,16 +391,68 @@
     "TIE_FORMAT": "무승부",
+    "FINISH_FORMAT": "#[yellow]@[] of [yellow]@[]",
+    "RATING_CALCULATION_NEXT_TEAM_ELIMINATION_WARNING": "[red]Your rating change will be calculated when some next team will be eliminated",
+    "RATING_CHANGE": "Your rating: [orange]@->@[white] @",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -278,10 +461,65 @@
+    "TIEBREAKER_FORMAT_FULL": "@[yellow]\u26A1[red]@[yellow]\u26A1\n@\n\n[white]@",
+    "SPECTATOR_STATS_PLAYER_FORMAT": "@ [yellow]@[white] @",
+    "SPECTATOR_STATS_FORMAT": "@\n@\n@\n@",
+    "NOTHING_TO_WAIT": "[scarlet]Nothing to wait for",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -278,10 +461,65 @@
+    "ADMIN_FORCED_TUTORIAL": "[green]서버가 처음이고 관리자가 튜토리얼을 강제 적용한 것 같습니다 .\n\n이 과정은 짧고, 그곳에서 당신은 간단한 것들을 배울 수 있습니다.\n\n10분 정도밖에 안걸립니다.",
+    "PLAYER_ABANDONED": "플레이어 @[white] from team [#@]@[]은 대결을 떠났고 다시는 돌아오지 않습니다."
+  },
+  "tutorial": {
```
Translate or exclude this whole section.

LeoSko commented on ko/Messages.json:
```diff
@@ -278,10 +461,65 @@
+    "DRAW_USAGE": "[yellow]/draw[] 를 입력하여 무승부를 받아들이기.",
+    "RECONNECTED_AFTER_ABANDONED": "경기를 포기한 후 다시 연결되었습니다. 계속 관전자로로 시청하거나 새로운 대결을 시작할 수 있습니다.",
+    "ADMIN_FORCED_TUTORIAL": "[green]서버가 처음이고 관리자가 튜토리얼을 강제 적용한 것 같습니다 .\n\n이 과정은 짧고, 그곳에서 당신은 간단한 것들을 배울 수 있습니다.\n\n10분 정도밖에 안걸립니다.",
+    "PLAYER_ABANDONED": "플레이어 @[white] from team [#@]@[]은 대결을 떠났고 다시는 돌아오지 않습니다."
```
Is "from team" translatable?

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "CHAT_PREFIX_SPECTATORS": "[@]관전자[]:",
+    "CHAT_PREFIX_TEAM_ONLY_FORMAT": "([#@]@[]):",
+    "CHAT_PREFIX_GROUP_ONLY_FORMAT": "([@]그룹원[]):",
+    "HISTORICAL_MATCH_ENTRY": "Match#@ @ [yellow]@\n[white]@\n@",
```
Translate or exclude.

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "ACHIEVEMENT_DESCRIPTION_PIECE_OF_A_CAKE": "서버의 생일을 축하하는 동안 대결을 진행하세요",
+    "BALANCE_UNITS": "[yellow]모든 유닛 공통[]\n- 각 유닛의 [green]최대[]사거리에서 발사합니다\n- 유닛의 체력이 일정 비율 이하로 떨어질 경우, 가까운 수리 지점\uF848으로 [green]자동 귀환[]합니다 \n 기준: T1-70%, T2-60%, T3-50%, T4-40%, T5-30%\n\n[yellow]지상 유닛 공통[]\n- 대상보다 자신의 사거리가 길 경우, 적을 쏠 때 움직이지 않습니다\n- 이 기능은 메이스\uF7FF, 펄서\uF7FC, 퀘이사\uF7FB, 아트락스\uF7F9, 스피록트\uF7F8에게는 적용되지 않습니다 \n- 노바 계열 유닛은 깊은 물에 빠지지 않고 비행하여 건너갑니다\n\n[yellow]비행 유닛 공통[]\n- 공격 대상을 상대로 빙빙 ...
+    "BALANCE_STATS": "[yellow]능력치[]\n체력 변경점은 데미지 배수를 설정된 데미지 배수를 거치고 적용됩니다.  (로직 역호환 가능)\n\n대거\uF800: 재장전 14⇨13\n메이스\uF7FF: [red]♥[]500⇨650, 30⇨34, 재장전 14⇨11, [yellow]\uE86B[]4⇨7\n포트리스\uF7FE: [red]♥[]800⇨1000, 54⇨62\n노바\uF7FD: [red]♥[]120⇨160, 14⇨16\n퀘이사\uF7FB: [red]♥[]650⇨700\n크롤러\uF7FA: [red]♥[]180⇨220, 85⇨90, [green]⛷[]1⇨1.05\nAtrax\uF7F9: reload 10⇨9, [green]⛷[]0.5⇨0.8\n스피록트\uF7F8: 재장전 15 ...
+    "BALANCE_BLOCKS": "[yellow]건설 블록[]\n20부터 시작해서, 건설 진행도에 비례해서 체력이 증가합니다   (기본 버전은 모든 과정에서 체력이 20).\n\n[yellow]보호막 프로젝터[]\n- 낙파 피해량을 33%까지 감소시키고, 보호막과 블록이 피해량을 나눠 받습니다\n- 건물에 피해를 입히는 대신 보호막이 덮인 건물 위로 발사되는 총알을 흡수합니다.\n- 아이템 폭발 대미지를 1/5로 감소시킵니다\n\n[yellow]코어[]\n- 기반 코어\uF868 [red]♥[]3500⇨3700\n- 조각 코어\uF869 [red]♥[]1100⇨1500\n\n모든 탄환은 5배의 넉백을 가집니다\n\n패럴랙스\uF801는 4배 더 강하게 끌어옵니다\n\n[yellow]상태이상[]\n[ ...
```
Here just a little thing left. :)

Nunting replied:
I'm not sure what that part means

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "ACHIEVEMENT_NAME_PIECE_OF_A_CAKE": "케이크 한 조각",
+    "ACHIEVEMENT_DESCRIPTION_PIECE_OF_A_CAKE": "서버의 생일을 축하하는 동안 대결을 진행하세요",
+    "BALANCE_UNITS": "[yellow]모든 유닛 공통[]\n- 각 유닛의 [green]최대[]사거리에서 발사합니다\n- 유닛의 체력이 일정 비율 이하로 떨어질 경우, 가까운 수리 지점\uF848으로 [green]자동 귀환[]합니다 \n 기준: T1-70%, T2-60%, T3-50%, T4-40%, T5-30%\n\n[yellow]지상 유닛 공통[]\n- 대상보다 자신의 사거리가 길 경우, 적을 쏠 때 움직이지 않습니다\n- 이 기능은 메이스\uF7FF, 펄서\uF7FC, 퀘이사\uF7FB, 아트락스\uF7F9, 스피록트\uF7F8에게는 적용되지 않습니다 \n- 노바 계열 유닛은 깊은 물에 빠지지 않고 비행하여 건너갑니다\n\n[yellow]비행 유닛 공통[]\n- 공격 대상을 상대로 빙빙 ...
+    "BALANCE_STATS": "[yellow]능력치[]\n체력 변경점은 데미지 배수를 설정된 데미지 배수를 거치고 적용됩니다.  (로직 역호환 가능)\n\n대거\uF800: 재장전 14⇨13\n메이스\uF7FF: [red]♥[]500⇨650, 30⇨34, 재장전 14⇨11, [yellow]\uE86B[]4⇨7\n포트리스\uF7FE: [red]♥[]800⇨1000, 54⇨62\n노바\uF7FD: [red]♥[]120⇨160, 14⇨16\n퀘이사\uF7FB: [red]♥[]650⇨700\n크롤러\uF7FA: [red]♥[]180⇨220, 85⇨90, [green]⛷[]1⇨1.05\nAtrax\uF7F9: reload 10⇨9, [green]⛷[]0.5⇨0.8\n스피록트\uF7F8: 재장전 15 ...
```
I'll accept `BALANCE_` translations for now, but keep in mind they change frequently. Next season I'll have to remove translation to make it actual.

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "NAME_NOT_ALLOWED_PREFIX": "당신의 이름\n[red]@[]\n은 허용되지 않았습니다. 다른 이름을 선택하세요.",
+    "CHAT_PREFIX_ALL": "[@]모두[]:",
+    "CHAT_PREFIX_SPECTATORS": "[@]관전자[]:",
+    "CHAT_PREFIX_TEAM_ONLY_FORMAT": "([#@]@[]):",
```
Do not translate formats, which in fact don't have anything to translate.

LeoSko commented on ko/Messages.json:
```diff
@@ -201,50 +224,158 @@
+    "SERVER_NAME_2V2": "[yellow]⚡ [red]Ranked [blue]2 vs 2 [yellow]⚡",
+    "SERVER_NAME_3V3": "[yellow]⚡ [red]Ranked [purple]3 vs 3 [yellow]⚡",
+    "SERVER_NAME_HEX": "[yellow]⚡ [red]Ranked [green]H E X [yellow]⚡",
+    "SERVER_DESCRIPTION": "[red]매치메이킹 1 vs 1\n[#ff00ff]시즌 VIII",
```
Wrong color of the season. Use `[#87ceeb]`

LeoSko reviewed (changes requested):
That's a great contribution! Please take a bit more time on highlighted issues and I'll deploy within an hour.

Nunting committed:
Update Messages.json

I left balance-effect empty because I thought it would be better to know in detail how it actually applies and translate it.
I revised the parts pointed out and some translations.
Adding tutorial translation.
